### ISSUE TITLE
Unminified source for annotorious.okfn.0.3.js

### ISSUE THREAD

**Issue by itskingori** [annotorious/annotorious-v1#121]
Is it possible to get this? I was able to download the Annotorious plugin for Annotator from:
- http://annotorious.github.io/plugins.html
- http://annotorious.github.io/plug-outs/okfn-annotator.html

**Comment by tsi** [comment]
Ever found it? Wanted to use it as a reference for my own plugin...

**Comment by itskingori** [comment]
@tsi Nope. I'd presume it's what's in the `\src` folder but I'm not familiar to the tooling (how to build etc.)  ...

**Comment by notbrain** [comment]
I know it's been a while but I just built annotorious with the plovr jar from another project...if you can get me the 0.3 source code I could try and build it for you. If you can use the latest version I can easily provide that, but didn't see a tag at 0.3 - just 0.2 and then 0.4.

Download plovr.jar from https://github.com/bolinfest/plovr and put in plovr/plovr4.jar then run the command (inside a new checkout of annotorious):

`java -jar plovr/plovr4.jar build standalone.json > annotorious.raw.js`

with standalone.json modified to RAW mode:

```
{
  "id": "annotorious",
  "define": {
    "goog.DEBUG": false,
    "goog.dom.ASSUME_STANDARDS_MODE": true
  },
  "inputs": [
    "src/annotorious.js",
    "src/api.js"           // exports JS API and sets up the plugin namespace
  ],
  "paths": [
    "src",
    "templates"
  ],
  "externs": [
    "externs/api.externs.js",
    "externs/openlayers.externs.js",
    "externs/openseadragon.externs.js",
    "externs/jquery.externs.js"
    //"json.js",
    //"webkit_console.js"
  ],
  "mode": "RAW",      // "RAW" or "ADVANCED"
  "level": "DEFAULT",      // "DEFAULT" or "VERBOSE"
  "pretty-print": true,
  "debug": false
}
```
